upgrade/helpers/docker.py

```diff
@@ -105,3 +105,25 @@ def docker_cleanup_containers():
     run('docker ps -a | grep \'days ago\' | awk \'{print $1}\' | xargs '
         '--no-run-if-empty docker rm ')
     logger.info('Cleaning UP of Docker containers ENDS')
+
+
+def docker_wait_until_repo_list(docker_id, timeout=5):
+    """Waits until the ak is attached and repo's are listed
+
+    :param string docker_id: ID of the docker container
+    :param int timeout: The polling timeout in minutes.
+    """
+    timeup = time.time() + int(timeout) * 60
+    while True:
+        result = docker_execute_command(
+            docker_id,
+            'yum repolist | grep repolist'
+        )
+        result = int(result.values()[0].split(':')[1].strip())
+        if time.time() > timeup:
+            logger.warning('There is no repos on {0}'.format(docker_id))
```

Comment:
The reason of not getting repos could be different ... so just say that timeout occurred. Unless you are not sure that there are really not repos.

If you still want to stick to it just do this -> s/is/are